CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Internal: Added test suite
```

Comment:
are we using `Internal:`?

Reply:
I got that from https://github.com/mineiros-io/terraform-google-project-iam/blob/main/CHANGELOG.md

Reply:
nice, will use it from now on!